test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
delete this file

Reply:
![Screenshot from 2021-09-01 21-06-20](https://user-images.githubusercontent.com/87750131/131676562-dff39b95-c41d-4be7-b4ee-d99784681296.png)